getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -756,22 +888,29 @@ return early with the ``return`` keyword, but they can't return any value.
 Referencing functions
 ^^^^^^^^^^^^^^^^^^^^^
 
-Contrary to Python, functions are *not* first-class objects in GDScript. This
-means they cannot be stored in variables, passed as an argument to another
-function or be returned from other functions. This is for performance reasons.
+Functions are first-class items in terms of the Callable object. Referencing a
+function by name without calling it will automatically generate the proper
+callable. This can be used to pass functions as arguments.
 
-To reference a function by name at run-time, (e.g. to store it in a variable, or
-pass it to another function as an argument) one must use the ``call`` or
-``funcref`` helpers::
+::
 
-    # Call a function by name in one step.
-    my_node.call("my_function", args)
+    func map(arr: Array, function: Callable) -> Array:
+        var result = []
+        for item in arr:
+            result.push_back(function.call(item))
+        return result
 
-    # Store a function reference.
-    var my_func = funcref(my_node, "my_function")
-    # Call stored function reference.
-    my_func.call_func(args)
+    func add1(value: int) -> int:
+        return value + 1;
 
+    func _ready() -> void:
+        var my_array = [1, 2, 3]
+        var plus_one = map(my_array, add1)
+        print(plus_one) # Prints [2, 3, 4].
```

Comment:
There should be 2 lines of space between those methods.

Reply:
Using two lines for separation on small doc snippets isn't helpful. I understand the style guide says so, but in small snippets it just looks off. That's why I asked an exception on documentation pages: https://github.com/godotengine/godot-docs/issues/3613#issuecomment-636367359

Reply:
Yes, the style *guide* shouldn't *mandate* style.